src/components/Accordion/AccordionItem/AccordionItem.jsx

```diff
@@ -7,28 +7,33 @@ import {AccordionContext} from '~/components/Accordion/Accordion.context';
 
 export const AccordionItem = ({id, label, icon, onClick, children, className, ...props}) => {
     const context = useContext(AccordionContext);
+    const [cssExpanded, setCssExpanded] = useState(null);
     const open = context.currentItem === id;
 
     const handleClick = (e, open) => {
         onClick(e, !open);
         context.onSetOpenedItem(id);
     };
 
+    useEffect(() => {
+        setCssExpanded({[styles.accordionItem_opened]: open});
```

Comment:
I don't understand the need of a state here - why don't we use directly open ?

Reply:
It's to trigger the animation after the render. We had some user feedback than they don't understand there is some accordion at the bottom of the page. We think to animate the opening of the accordion can improve this behavior.

Reply:
So it's only for the first render, if the accordion item is directly opened ? This has no impact when opening/closing the accordion after its first render ?